### PR TITLE
styles / remove extraneous querySelector

### PR DESCRIPTION
**Pull request by bmeck**
notable changes:

* added auto evaluate after change feature
* some responsive layout for smaller devices
* moved color to JS Yellow and sans-serif to match theme of JS
* move to github ribbon rather than link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,45 +1,49 @@
...
+        loading.close();
+      });
+    </script>
+    <!-- github ribbon -->
+    <a id="github" href="https://github.com/engine262/engine262"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1"></a>
     <h1>engine262</h1>
-    <h2>A self-hosted JavaScript engine for development and exploration.</h2>
-    <a href="https://github.com/engine262/engine262">engine262 on GitHub</a>
+    <p>A self-hosted JavaScript engine for development and exploration.</p>
+    <!--<a href="https://github.com/engine262/engine262">engine262 on GitHub</a>-->
```
Comment (devsnek) [inline index.html]:
```suggestion

```

File: index.html [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
+      Loading...
+    </dialog>
+    <script>
+      loading.showModal();
+      window.addEventListener('error', () => {
+        loading.close();
+      });
+    </script>
+    <!-- github ribbon -->
+    <a id="github" href="https://github.com/engine262/engine262"><img width="149" height="149" src="https://github.blog/wp-content/uploads/2008/12/forkme_right_darkblue_121621.png?resize=149%2C149" class="attachment-full size-full" alt="Fork me on GitHub" data-recalc-dims="1"></a>
```
Comment (devsnek) [inline index.html]: Can this be an embedded SVG or something? i think http://tholman.com/github-corners/ is where people get ribbons these days.

File: index.html [hunk 3]
```diff
@@ -1,45 +1,49 @@
...
-    <a href="https://github.com/engine262/engine262">engine262 on GitHub</a>
+    <p>A self-hosted JavaScript engine for development and exploration.</p>
+    <!--<a href="https://github.com/engine262/engine262">engine262 on GitHub</a>-->
 
-    <p>Features</p>
-    <ul id="features"></ul>
+    <main>
+      <section id="input-section">
+        <fieldset><legend>Features</legend>
+          <ul id="featureList"><li><label><input id="autoevaluate" checked type="checkbox">Auto Evaluate</label></li></ul>
```
Comment (devsnek) [inline index.html]: can auto evaluate be separate graphically from the engine features?

I'd probably move auto evaluate, script/module, and the evaluate button to one row.

Comment (devsnek) [reply index.html]: also, auto evaluate isn't clearing the console.

File: index.html (hunk 2 above)
Comment (bmeck) [reply index.html]: I'm neutral but kind of like that this one at least says in text what it is about to do.

File: index.html (hunk 3 above)
Comment (bmeck) [reply index.html]: sure, but evaluate should probably be its own row since it is not a setting but an action.

File: index.html (hunk 2 above)
Comment (devsnek) [reply index.html]: copying the png into the repo would also work i guess. i just don't want to be loading things off of random sites.

File: index.html (hunk 3 above)
Comment (bmeck) [reply index.html]: moved mode and auto evaluate together and fixed clearing output

File: worker.js [hunk 4]
```diff
@@ -34,7 +34,18 @@ addEventListener('message', ({ data }) => {
...
+    // `);
+    const err = new Value(realm, (args) => {
+      postMessage({ error: args.map((a) => inspect(a)) });
+      return Value.undefined;
+    }, [], realm);
+    // err.Value.Call(null, (...args) => {
+    //   postMessage({ log: args.map((a) => inspect(a)) });
+    // });
     Abstract.CreateDataProperty(realm.global, new Value(realm, 'print'), print);
+    Abstract.CreateDataProperty(realm.global, new Value(realm, 'err'), err);
```
Comment (devsnek) [inline worker.js]: unrelated changes?

File: index.html (hunk 2 above)
Comment (bmeck) [reply index.html]: vendored into `github_ribbon.png`

File: worker.js (hunk 4 above)
Comment (bmeck) [reply worker.js]: doh, yes sorry

**Comment by bmeck** [comment]
squashed

**Comment by devsnek** [comment]
thanks bradley :heart:
